### ISSUE TITLE
passing fbx path doesn't allow for in-place cconvertion

### ISSUE THREAD

**Issue by GoncaloBastos** [BobbyAnguelov/FbxFormatConverter#3]
I'm trying to run a shell script that uses the query option to filter out non-ascii fbxs in order to optimize our conversion times, but for some reason, if I try to call it without the -o option specified, it throws an error: 

`Error! File export failed: - File not created`

However, if I try to make the exact same call but pass a folder instead of a specific file, then it manages to replace the existing ascii files with the binary ones without issue. Is there a way to get single files converted in place?

**Comment by BobbyAnguelov** [comment]
Are you trying to convert or are you trying to query? The query only takes -q and nothing else.

If you are trying to convert in place, there might be an issue which I can try to check.

Can you explain exactly what you're doing? Are you querying or converting?

**Comment by GoncaloBastos** [comment]
First I use query in our asset folder to fetch the fbx that are ascii.

I then use the result of this query to convert the ascii fbx into binary in place.

Here's my script:
```
for f in **/*.fbx; do
	output="$(./.FbxFormatConverter.exe -q "$f" | awk -F" - " '/ascii/{print $1}')"
	if [[ -n $output ]]
	then
		echo "Converting $output to binary."
		./.FbxFormatConverter.exe -c "$output" -binary
	fi
done
```

**Comment by GoncaloBastos** [comment]
any suggestions?

**Comment by GoncaloBastos** [comment]
@BobbyAnguelov ?

**Comment by BobbyAnguelov** [comment]
Should be fixed by 8ac2451
